popup.js

```diff
@@ -78,6 +78,18 @@ document.getElementById('convertto').onchange=function (e) {
 				}
 			}
 			fset.innerHTML = fdef;
+			var customFields = [...document.querySelectorAll('.slds-input')]
+			customFields.forEach((textfield)=> textfield.addEventListener('change',(e)=>{
+				chrome.storage.sync.set({'fieldValue':e.target.value});
+			}));
+			chrome.storage.sync.get("fieldValue",({fieldValue})=>{
+				if(fieldValue==undefined ){
```

Comment:
this IF structure is too complex. it should be like this

```
if (fieldValue!=undefined) {
do whatever
}
```

the `return` you are using there adds no important information, and makes the code harder to read and maintain

Reply:
i get it